sender.go

```diff
@@ -342,7 +343,12 @@ func (s *LineSender) Long256Column(name, val string) *LineSender {
 		return s
 	}
 	s.buf.WriteByte('=')
-	s.lastErr = s.writeStrValue(val, false)
+	hexVal := "0x"
+	if utf8.RuneCountInString(val.Text(16))%2==1 {
```

Comment:
This check is quite expensive and I'm not sure that I understand its purpose. As a rule of thumb, we don't want any heap allocations in `Long256Column` calls.

Reply:
The rune count check is necessary because the user might enter odd values like `fff` instead of `0fff` which would be accepted by the system.
Hence this check is added. We can either optimise it or mention it in the documentation. But it is necessary.

Reply:
The final output of `0xfffi` is `0x0fff`, this same value is also needed to pass the tests. Hence we need to add zero.

Reply:
`0x0fff` is the output format returned when querying the database, not the input format. There is no need to add the extra `0` to the input.